salesapp.js

```diff
@@ -1,205 +1,105 @@
-  }
-};
-locSeattleCenter.calcCookies = function() {
-  for(var i = 0; i < storeHours.length; i++){
-    this.cookiesPerHour[i] = Math.floor(this.rndmCustPerHour[i] * this.avgCookiesPerCust);
+    this.rndmCustPerHour[i] = Math.floor(Math.random() * (this.maxCustPerHour - this.minCustPerHour + 1) + this.minCustPerHour);
   }
 };
-locCapitalHill.calcCookies = function() {
-  for(var i = 0; i < storeHours.length; i++) {
-    this.cookiesPerHour[i] = Math.floor(this.rndmCustPerHour[i] * this.avgCookiesPerCust);
-  }
-};
-locAlki.calcCookies = function() {
+
+CookieStore.prototype.calcCookiesSoldPerHour = function() {
+  this.genRndmCust();
   for(var i = 0; i < storeHours.length; i++) {
-    this.cookiesPerHour[i] = Math.floor(this.rndmCustPerHour[i] * this.avgCookiesPerCust);
+    this.cookiesSoldPerHour[i] = Math.floor(this.rndmCustPerHour[i] * this.avgCookiesPerCust);
+    this.totalDailyCookies += this.cookiesSoldPerHour[i];
   }
 };
 
-//caltulate total daily cookies sold using cookies sold per hour and += incrementing totalDailyCookies for each # in that array
-locFirstAndPike.calcDailyCookies = function() {
-  for(var i = 0; i < this.cookiesPerHour.length; i++) {
-    this.totalDailyCookies += this.cookiesPerHour[i];
-  }
-};
-locSeaTacAirport.calcDailyCookies = function() {
-  for(var i = 0; i < this.cookiesPerHour.length; i++) {
-    this.totalDailyCookies += this.cookiesPerHour[i];
-  }
-};
-locSeattleCenter.calcDailyCookies = function() {
-  for(var i = 0; i < this.cookiesPerHour.length; i++) {
-    this.totalDailyCookies += this.cookiesPerHour[i];
-  }
-};
-locCapitalHill.calcDailyCookies = function() {
-  for(var i = 0; i < this.cookiesPerHour.length; i++) {
-    this.totalDailyCookies += this.cookiesPerHour[i];
-  }
-};
-locAlki.calcDailyCookies = function() {
-  for(var i = 0; i < this.cookiesPerHour.length; i++) {
-    this.totalDailyCookies += this.cookiesPerHour[i];
-  }
-};
+var tableEl = document.getElementById('cookieTable');
 
-//function to populate list here
-locFirstAndPike.outputToSales = function() {
-  var puttingHere = document.getElementById('firstAndPikeList');
-  for(var i = 0; i < storeHours.length; i++) {
-    var htmlTag = document.createElement('li');
-    htmlTag.textContent = storeHours[i] + ': ' + locFirstAndPike.cookiesPerHour[i] + ' cookies';
-    puttingHere.appendChild(htmlTag);
-  }
-  var htmlTag = document.createElement('li');
-  htmlTag.textContent = 'Total: ' + locFirstAndPike.totalDailyCookies + ' cookies';
-  puttingHere.appendChild(htmlTag);
-};
-locSeaTacAirport.outputToSales = function() {
-  var getElement = document.getElementById('seaTacList');
+CookieStore.prototype.renderTable = function(rowCounter) {
+  this.calcCookiesSoldPerHour();
+  var trEl = document.createElement ('tr');
+  var tdEl = document.createElement ('td');
+  tdEl.textContent = this.storeName;
+  trEl.appendChild(tdEl);
+  var tdEl = document.createElement ('td');
```

Comment:
As long as you've appended the last `<td>` element you created, you don't have to re-declare the `tdEl` variable. You can just re-assign it. So, no need for the extra `var`